spec/integration_spec.rb

```diff
@@ -1399,6 +1399,8 @@ class SerializedDocument < ActiveRecord::Base
 end
 
 describe 'Raise on failure' do
+  before { Vegetable.instance_variable_set('@ms_indexes', nil) }
```

Comment:
Required because I'm messing with the internals of that class when I'm adding/removing the mocks to raise the `TimeoutError`.